README.md

```diff
@@ -48,10 +48,9 @@ structures used.
 <a name="HUJIex"/>
 
 #### Loading data offline ####
-it's recommended to load the data of all courses and tracks, rather than download and re-parse thousands of files. 
-1. Download the [7z dump file](https://drive.google.com/file/d/1TQSQ--VWFKt0CCZlBOZRUxKuH6Y6u0DL/view?usp=sharing) (HUJI account required)
-2. Extract it into the `Parser` folder
-3. Run `OfflineParser.py -> load_all_dumped()` _(Tip: if fetching corner-stone coursers gets stuck, comment out that line and continue with out that data, there's an [open issue](https://github.com/ScDor/Closure/issues/4))_
+
+Upon migrating the database, a data dump containing all parsed course and track data will be
+downloaded from the internet and inserted into the database.
```

Comment:
Say someone forks this, will they download the data from our host as well?

Reply:
just saw it's from Github, never mind

Reply:
Actually, although I've attached a zip-file to the repo, the migration currently loads the zip from a public GCP bucket.
Do you have any opinions on which method is preferable? 

As I see it:

- on the one hand, a zipfile in the repo is version controlled
- on the other hand, one day we might want to decouple the parser and the parsed artifacts from the backend repo, and in that case we'd need to download it remotely from some server anyway

Reply:
I see version control and the relative simplicity as a slight advantage of github. Should we want to change it later (decoupling etc), it shouldn't be a big issue anyway.

Reply:
Understandable, I'll switch it to use the zip file